src/Umbraco.Web.UI.Client/src/views/components/property/umb-property.html

```diff
@@ -1,13 +1,16 @@
 <div class="umb-property">
     <ng-form name="propertyForm">
-        <div class="control-group umb-control-group" ng-class="{hidelabel:property.hideLabel}" >
-            
+        <div class="control-group umb-control-group" ng-class="{hidelabel:property.hideLabel}">
+
             <val-property-msg property="property"></val-property-msg>
-            
+
             <div class="umb-el-wrap">
-                
+
                 <label class="control-label" ng-hide="property.hideLabel" for="{{property.alias}}" ng-attr-title="{{propertyAlias}}">
                     {{property.label}}
+                    <strong ng-if="property.validation.mandatory">
+                        <span class="red">&nbsp;*&nbsp;</span>
```

Comment:
Suggestion: If there's a better/more suitable class name in the Umbraco UI stylesheets than just `red`, it might be better to use that? (Like `umb-required` or `umb-property-required` etc.)